components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -480,9 +484,11 @@
                             if (propertyName != null && propertyName.trim().length() > 0) {
 
                         %>
-                        <td class="leftCol-med" width="50%" style="text-align:left;"><%=propertyName%><span
-                                class="required">*</span></td>
-                        <input type="hidden" name=<%=name%> id=<%=name%> value="<%=mandatories[j].getName()%>"/>
+                        <td class="leftCol-med" width="50%" style="text-align:left;"><%=Encode.forHtml(propertyName)%>
+                        <span class="required">*</span></td>
+                        <input type="hidden" name=<%=Encode.forHtmlAttribute(name)%>
```

Comment:
here variable name has a hardcoded value which does not contain special characters. Thus, not necessary to encode